test-cases/longevity/longevity-10gb-3h-schema-changes.yaml

```diff
@@ -0,0 +1,24 @@
+test_duration: 255
+stress_cmd: ["cassandra-stress write cl=QUORUM duration=180m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=1000 -pop seq=1..10000000 -log interval=5"
+             ]
+
+n_db_nodes: 6
+n_loaders: 2
+n_monitor_nodes: 1
+
+instance_type_db: 'i3.4xlarge'
+gce_instance_type_db: 'n1-highmem-16'
+gce_instance_type_loader: 'e2-standard-4'
+azure_instance_type_db: 'Standard_L8s_v2'
+run_fullscan: '{"ks_cf": "keyspace1.standard1", "interval": 5}' # 'ks.cf|random, interval(min)'
+nemesis_class_name: 'SisyphusMonkey'
+nemesis_seed: '111'
+nemesis_interval: 5
+nemesis_include_filter: ['schema_changes']
+
+user_prefix: 'longevity-10gb-3h'
+space_node_threshold: 64424
+
+gce_n_local_ssd_disk_db: 2
+use_preinstalled_scylla: true
+run_db_node_benchmarks: true
```

Comment:
I might consider remove this, mesuring the same instance types, won't teach us anything new (and would waste some of that startup time)

Reply:
Not related to this PR, for now I'm using this longevity just to check it as a reproducer for an issue.
BUT, we thought making this one as true by default.